models/models_methods.go

```diff
@@ -864,6 +952,34 @@ func MergeStepWith(step, otherStep stepmanModels.StepModel) (stepmanModels.StepM
 	return step, nil
 }
 
+// ----------------------------
+// --- WorkflowIDData
+
+// GetWorkflowIDWorkflowDataPair ...
+func GetWorkflowIDWorkflowDataPair(workflowListItem WorkflowListItemModel) (string, WorkflowModel, error) {
+	if len(workflowListItem) > 1 {
+		return "", WorkflowModel{}, errors.New("WorkflowListItem contains more than 1 key-value pair")
+	}
+	for key, value := range workflowListItem {
+		return key, value, nil
+	}
+	return "", WorkflowModel{}, errors.New("WorkflowListItem does not contain a key-value pair")
+}
+
+// ----------------------------
+// --- StageIDData
+
+// GetStageIDStageDataPair ...
+func GetStageIDStageDataPair(stageListItem StageListItemModel) (string, StageModel, error) {
```

Comment:
I would have the same question as for `GetWorkflowIDWorkflowDataPair`, where are we using the returned `StageModel`?

Reply:
Same as above.